admin/migrate-data.ps1

```diff
@@ -78,54 +83,76 @@ Write-Verbose 'Checking PATH prerequisites...'
+	if ($LASTEXITCODE -ne 0) {
+		throw 'Unable to create database'
+	}
 
-Write-Verbose "Creating database named $dbName..."
-docker exec $dbContainerName mysql -uroot --password="$dbRootPwd" -e "CREATE DATABASE $dbName"
-if ($LASTEXITCODE -ne 0) {
-	throw 'Unable to create database'
-}
+	Write-Verbose 'Creating temporary directory...'
+	docker exec $dbContainerName mkdir -p /tmp/srm
+	if ($LASTEXITCODE -ne 0) {
+		throw 'Unable to create directory'
+	}
 
-Write-Verbose 'Creating temporary directory...'
-docker exec $dbContainerName mkdir -p /tmp/codedx
-if ($LASTEXITCODE -ne 0) {
-	throw 'Unable to create directory'
-}
+	Write-Verbose 'Copying database dump file to container...'
+	docker cp $dbDumpFilePath $dbContainerName`:/tmp/srm/dump-srm.sql
+	if ($LASTEXITCODE -ne 0) {
+		throw 'Unable to copy dump file to directory'
+	}
 
-Write-Verbose 'Copying database dump file to container...'
-docker cp $dbDumpFilePath $dbContainerName`:/tmp/codedx/dump-codedx.sql
-if ($LASTEXITCODE -ne 0) {
-	throw 'Unable to copy dump file to directory'
-}
+	Write-Verbose 'Importing database dump file (may take a while)...'
+	docker exec $dbContainerName "bash" "-c" "mysql -uroot --password=""$dbRootPwd"" $dbName < /tmp/srm/dump-srm.sql"
+	if ($LASTEXITCODE -ne 0) {
+		throw 'Unable to import database dump file'
+	}
 
-Write-Verbose 'Importing database dump file (may take a while)...'
-docker exec $dbContainerName "bash" "-c" "mysql -uroot --password=""$dbRootPwd"" $dbName < /tmp/codedx/dump-codedx.sql"
-if ($LASTEXITCODE -ne 0) {
-	throw 'Unable to import database dump file'
-}
+	Write-Verbose 'Deleting database dump file...'
+	docker exec $dbContainerName rm -Rf /tmp/srm
+	if ($LASTEXITCODE -ne 0) {
+		throw 'Unable to delete database dump file'
+	}
+} else {
+	$externalDatabaseInstructions = @'
 
-Write-Verbose 'Deleting database dump file...'
-docker exec $dbContainerName rm -Rf /tmp/codedx
-if ($LASTEXITCODE -ne 0) {
-	throw 'Unable to delete database dump file'
+Since your SRM deployment uses an external database (one that you maintain on your own
+that is not installed or updated by the SRM deployment script), you must restore the dump-srm.sql file you
+created in Step 5 of the Migrate SRM Data to Docker Compose instructions.
+
+You can restore mysqldump files using a command that looks like this:
+
+  mysql -uroot -p srmdb < dump-srm.sql
+
+Note: Replace 'root' and 'srmdb' as necessary.
+'@
+	Write-Host $externalDatabaseInstructions
+	Read-Host -Prompt 'Restore your database (see above) and then press Enter to continue...'
 }
 
 Write-Verbose 'Deleting directories...'
 '/opt/codedx/analysis-files','/opt/codedx/keystore','/opt/codedx/mltriage-files' | ForEach-Object {
 
 	Write-Verbose "Deleting directory $_..."
-	docker exec $dbContainerName "bash" "-c" "if test -d $_; then rm -Rf $_; fi"
+	docker exec $tomcatContainerName "bash" "-c" "if test -d $_; then rm -Rf $_; fi"
```

Comment:
Just pointing out this line, I believe this had been using the wrong container since the db container does not have the appdata. The appdata volume is mounted on the tomcat container.